docs/Software Architecture Document/Components View/assets/ComponentView.gliffy

```diff
@@ -0,0 +1 @@
adowY":0.0}},"linkMap":[],"children":[],"hidden":false,"layerId":"Zf4l5osTBAgn"},{"x":1284.6579302399764,"y":2167.2388059701498,"rotation":0.0,"id":470,"width":100.0,"height":100.0,"uid":"com.gliffy.shape.uml.uml_v2.deployment.association","order":67,"lockAspectRatio":false,"lockShape":false,"constraints":{"constraints":[],"startConstraint":{"type":"StartPositionConstraint","StartPositionConstraint":{"nodeId":222,"py":0.5,"px":1.0}},"endConstraint":{"type":"EndPositionConstraint","EndPositionConstraint":{"nodeId":471,"py":0.5,"px":0.0}}},"graphic":{"type":"Line","Line":{"strokeWidth":1.0,"strokeColor":"#000000","fillColor":"none","dashStyle":null,"startArrow":0,"endArrow":0,"startArrowRotation":"auto","endArrowRotation":"auto","interpolationType":"linear","cornerRadius":null,"controlPath":[[48.25197708773794,-57.999999999999545],[191.6128537496445,-57.999999999999545],[334.97373041155106,-57.999999999999545],[478.3346070734576,-57.999999999999545]],"lockSegments":{},"ortho":true}},"linkMap":[],"children":[],"hidden":false,"layerId":"Zf4l5osTBAgn"},{"x":1297.885563804014,"y":1585.365369417042,"rotation":0.0,"id":502,"width":90.0,"height":41.0,"uid":"com.gliffy.shape.basic.basic_v1.default.text","order":68,"lockAspectRatio":false,"lockShape":false,"graphic":{"type":"Text","Text":{"overflow":"none","paddingTop":2,"paddingRight":2,"paddingBottom":2,"paddingLeft":2,"outerPaddingTop":6,"outerPaddingRight":6,"outerPaddingBottom":2,"outerPaddingLeft":6,"type":"fixed","lineTValue":null,"linePerpValue":null,"cardinalityType":null,"html":"<p style=\"text-align:center;\"><span style=\"font-weight:bold;font-size:18px;font-family:Arial;\"><span style=\"\"><span style=\"\">Secuirty</span><br /></span></span></p><p style=\"text-align:center;\"><span style=\"font-size:18px;font-family:Arial;\"><span style=\"font-weight:bold;\">Manager</span><span style=\"font-family:Courier;text-decoration:none;\"><br /></span></span></p>","tid":null,"valign":"middle","vposition":"none","hposition":"none"}},"linkMap":[],"children":[],"hidden":false,"layerId":"Zf4l5osTBAgn"},{"x":1267.885563804014,"y":1591.3237201914244,"rotation":0.0,"id":501,"width":30.0,"height":30.0,"uid":"com.gliffy.shape.uml.uml_v2.deployment.interface","order":69,"lockAspectRatio":true,"lockShape":false,"graphic":{"type":"Shape","Shape":{"tid":"com.gliffy.stencil.ellipse.basic_v1","strokeWidth":2.0,"strokeColor":"#000000","fillColor":"#FFFFFF","gradient":false,"dashStyle":null,"dropShadow":false,"state":0,"opacity":1.0,"shadowX":0.0,"shadowY":0.0}},"linkMap":[],"children":[],"hidden":false,"layerId":"Zf4l5osTBAgn"},{"x":172.966589598574,"y":1590.865369417042,"rotation":90.0,"id":506,"width":29.08329845123481,"height":30.0,"uid":"com.gliffy.shape.uml.uml_v2.component.required_interface","order":70,"lockAspectRatio":true,"lockShape":false,"graphic":{"type":"Shape","Shape":{"tid":"com.gliffy.stencil.circle.half_v1","strokeWidth":2.0,"strokeColor":"#000000","fillColor":"#FFFFFF","gradient":false,"dashStyle":nul
```

Comment:
@LuxoftAKutsan, I think `Components View` should look like this:
[ComponentsView.gliffy.txt](https://github.com/smartdevicelink/sdl_core_guides/files/696213/ComponentsView.gliffy.txt)
![untitled](https://cloud.githubusercontent.com/assets/18613689/21807991/e1ffdc10-d749-11e6-99b1-bb90a9bd4525.png)